Contributing.md

```diff
@@ -32,6 +32,12 @@ To test with the Client integration of Eclipse IDE:
   }
   ```
   - Click `Debug`
+- Connect Remote Java Application to debug Camel DAP server project
+  - Create a `Remote Java Application` configuration
+  - In `project` filed, set `camel-dap-server`
+  - In `port` filed, set `3000`
+  - Select `Allow termination of remote VM`
+  - Click `Debug`
```

Comment:
can be provided in a different PR

Reply:
ok 👍🏻 I will provide in another, no problem